galley/formatted_queries.py

```diff
@@ -283,25 +220,43 @@ def get_recipe_allergens(recipe_dietry_flags: List[Dict]) -> Dict:
         DietaryFlagEnum.SMOKED_MEATS.value: 'smoked meats',
         DietaryFlagEnum.BEEF.value: 'beef'
     }
-    allergens = []
+    allergens = [
+        allergen for df in dietry_flags
+        if (allergen:= allergen_labels.get(df.get('dietaryFlag', {}).get('id')))
```

Comment:
Adding a space around the assignment here
```suggestion
        if (allergen := allergen_labels.get(df.get('dietaryFlag', {}).get('id')))
```